api/src/main/java/org/opennms/newts/api/query/StandardAggregationFunctions.java

```diff
@@ -69,6 +71,45 @@ public Double apply(Collection<Double> input) {
             return min;
         }
 
-    }
+    },
+    P95 {
+
+        @Override
+        public Double apply(Collection<Double> input) {
```

Comment:
We should aggregate these function and pass the percentile as an argument instead of defining it twice.